packages/driver/src/cy/commands/actions/type.ts

```diff
@@ -173,7 +173,15 @@ export default function (Commands, Cypress, cy, state, config) {
     const win = state('window')
 
     const getDefaultButtons = (form) => {
-      return form.find('input, button').filter((__, el) => {
+      const formId = form.attr('id')
+      const nestedButtons = form.find('input, button')
+
+      const possibleDefaultButtons: JQuery<any> = formId ? $dom.wrap(_.uniq([
+        ...nestedButtons,
+        ...cy.$$('body').find(`input[form='${formId}'], button[form='${formId}']`),
```

Comment:
I wasn't sure what `cy.$$` is here.  Maybe we're looking for `$dom('body')` instead?

Reply:
Here's the definition for `cy.$$`: https://github.com/cypress-io/cypress/blob/6e03e020682e73e0e5480fe010fe818fce488205/packages/driver/src/cypress/cy.ts#L349-L355

So it seems like it is in effect doing `$dom('body')`.

To decrease the statefulness of this code, the context can be set to `Node.ownerDocument`:

```suggestion
        ...$dom.query('body', form.attr('ownerDocument')).find(`input[form='${formId}'], button[form='${formId}']`),
```

Reply:
Did you mean `form.prop('ownerDocument')` ?

Reply:
Yes, my bad.